engine/src/main/java/com/redhat/cloud/notifications/events/EventConsumer.java

```diff
@@ -226,12 +219,11 @@ public void process(String payload) {
                     TAG_KEY_APPLICATION, tags.getOrDefault(TAG_KEY_APPLICATION, ""),
                     TAG_KEY_EVENT_TYPE_FQN, tags.getOrDefault(TAG_KEY_EVENT_TYPE_FQN, "")
             ));
-            Log.debug("Processing Kafka message - end");
         }
+        return message.ack();
```

Comment:
Won't this lead to exceptions/disconnects if some endpoints take long to process?

Reply:
If it does, we will simply put the annotation back.

Reply:
I recall we had issues without the annotation `@Acknowledgment(PRE_PROCESSING)` - we should be extra careful with QA tests that might have tons of endpoints - I suppose that once we do the split it will be less trouble in that regard.